TezosKit/Client/PocketNetworkClient.swift

```diff
@@ -0,0 +1,101 @@
+// Copyright Keefer Taylor, 2019
+
+import Foundation
+import PocketSwift
+
+public class PocketNetworkClient: NetworkClient {
```

Comment:
Could you document this class? What is Pocket? Where would I go to grab a developer ID for instance? (I know the answers to these but the casual observer might not). 

Feel free to just link to your own documentation, or publish a markdown file if you'd like.